### DISCUSSION TITLE
没啥用的想法，关于头衔。

### DISCUSSION THREAD

**Discussion by xiamuguizhi** [ArtalkJS/Artalk#107] [Ideas]
就是一个没啥用的想法，你听听看就好了也不知道是不是累赘 哈哈。

就是 关于 站长 本来就有一个 评论区显示的头衔 `管理员` 。

假如我想给一些经常来访熟悉的朋友 加个  xxx头衔 呢， 感觉挺好玩的 哈哈。

当然这是一个没啥用的想法， 等以后完善了在考虑 考虑 哈哈 哈哈！！

![image](https://user-images.githubusercontent.com/19650297/163738351-3472c53f-9ef4-47e0-9e37-d22be1ca00b8.png)

**Comment by qwqcode** [comment]
虽然目前没有在前端实现这个功能，但你可以手动在数据库中修改，如图字段内容：

![image](https://user-images.githubusercontent.com/22412567/163757063-ab7bb5f8-0e00-49d2-ac88-a59e8ffa4311.png)

**Reply by xiamuguizhi** [reply]
懂了，懂了，感谢
